src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/auth/RegisterPage.java

```diff
@@ -1,15 +1,18 @@
-package com.wikia.webdriver.elements.mercury.components.signup;
-
+package com.wikia.webdriver.pageobjectsfactory.pageobject.auth;
 
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
 import com.wikia.webdriver.common.core.Assertion;
 import com.wikia.webdriver.common.core.configuration.Configuration;
 import com.wikia.webdriver.elements.mercury.components.TopBar;
-import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
-
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 
-public class RegisterArea extends WikiBasePageObject {
+public class RegisterPage extends BaseAuthPage {
+
+  public RegisterPage open() {
+      driver.get(urlBuilder.getUrlForWiki() + URLsContent.SPECIAL_USER_SIGNUP);
+      return this;
+  }
 
   @FindBy(css = ".auth.desktop.signin-page")
   private WebElement authModal;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1213)